### ISSUE TITLE
loss functions described in the paper and some questions about the implementation

### ISSUE THREAD

**Issue by kuoweilai** [SNU-LIST/QSMnet#2]
Hello, really appreciate your work, and I have several questions below.

1. I just wondering there seems no model loss and gradient loss in this repository. Are you planning to upload that part of code?

2. How many gpu do you use for training QSMnet for batch size 12?

3. It seems like you normalize the input and output twice(on the side of making patches and before training.)

Thank you for your time.

**Comment by wjjung93** [comment]
Sorry for late response.

1. Now model loss and gradient loss are available in Code/utils.py 

2. We used a single GPU for training, 1080 TI. 

3. Sorry for misleading. We revised training_data_patch.py code by removing normalization process. (Of course it should be done only once :)

Thanks for your feedback
